src/WordPress/JsonMapper/AnnotationMap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace WordPress\JsonMapper;
+
+class AnnotationMap
```

Comment:
Is this for a method? Property? An entire class? It seems to hold information about `var`, `params`, and `return`. It would be nice if there was a documentation block to explain the semantics.

Reply:
For our use cases `AnnotationMap` could've been dropped entirely. So that is was has happened now.